components/combobox/stories/combobox.stories.js

```diff
@@ -165,6 +171,20 @@ QuietGroup.parameters = {
 	chromatic: { disableSnapshot: true },
 };
 
+/**
+ * Combo boxes have a read-only option for when content in the disabled state still needs to be shown. This allows for content to be copied, but not interacted with or changed. A combo box does not have a read-only option if no selection has been made.
+*/
+
+export const ReadOnly = Template.bind({});
+ReadOnly.tags = ["!dev"];
+ReadOnly.args = {
+	isReadOnly: true,
+	value: "Ballard"
+};
+ReadOnly.parameters = {
+	chromatic: { disableSnapshot: true }
+};
```

Comment:
Let's add `ReadOnly.storyName = "Read-only";` so the story name is in the correct sentence-case.